src/FeatureFlagSwitchable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace SilverStripe\FeatureFlags;
+
+interface FeatureFlagSwitchable
```

Comment:
Comment this, something like: "implementations of this will handle the persistent modification of feature flag configuration"